modules/module_loan.rb

```diff
@@ -0,0 +1,22 @@
+require_relative 'module_data'
+
+# ModuleLoan module handles loan approval 
+module ModuleLoan
+  include ModuleData
+  
+  #display loan details for all countries
+  def loan_approve()
```

Comment:
```suggestion
  def loan_approve
```
@dinnupatel100 No paranthesis required if functional parameters are not present